run_action.py

```diff
@@ -229,6 +229,55 @@ def main():
         print("Warnings found but none in lines changed by this pull request.")
         return 0
 
+    # Load the existing review comments
+    existing_pull_request_comments = []
+    # Request a maximum of 100 pages (3000 comments)
+    for page_num in range(1, 100):
+        pull_request_comments_url = "%s/repos/%s/pulls/%s/comments?page=%d" % (
+            github_api_url,
+            repo,
+            args.pull_request_id,
+            page_num,
+        )
+        pull_request_comments_result = requests.get(
+            pull_request_comments_url,
+            headers={
+                "Accept": "application/vnd.github.v3+json",
+                "Authorization": "token %s" % github_token,
+            },
+        )
+
+        if pull_request_comments_result.status_code != requests.codes.ok:
+            print(
+                "Request to get pull request comments failed with error code: "
+                + str(pull_request_comments_result.status_code)
+            )
+            return 1
+
+        pull_request_comments = json.loads(pull_request_comments_result.text)
+
+        if len(pull_request_comments) == 0 :
+            break
+
+        existing_pull_request_comments = existing_pull_request_comments + pull_request_comments
```

Comment:
Minor comment for a future pull request or... never: Can become `existing_pull_request_comments += pull_request_comments`

Reply:
Sorry for this. You don't always notice such things right away :)